src/website/app/demos/Tabs/examples/Tab/icons.js

```diff
@@ -1,10 +1,54 @@
 /* @flow */
+import React from 'react';
 import IconCameraAlt from 'mineral-ui-icons/IconCameraAlt';
 import IconInfoOutline from 'mineral-ui-icons/IconInfoOutline';
 import IconMusicNote from 'mineral-ui-icons/IconMusicNote';
 import Tabs, { Tab } from '../../../../../../library/Tabs';
 import Text from '../../../../../../library/Text';
-import content from '../../shared/content';
+import DemoLayout from '../../../shared/DemoLayout';
+
+const content = {
+  bio: `The Rolling Stones are an English rock band formed in London, England, in 1962. The first stable line-up consisted of Brian Jones (guitar, harmonica), Mick Jagger (lead vocals), Keith Richards (guitar, backing vocals), Bill Wyman (bass), Charlie Watts (drums), and Ian Stewart (piano). Stewart was removed from the official line-up in 1963 but continued as a touring member until his death in 1985. Brian Jones was the original leader of the group.`,
+  albums: `
+  The Rolling Stones (1964)
+  12 X 5 (1964)
+  The Rolling Stones No. 2 (1965)
+  Out of Our Heads (1965)
+  December's Children (And Everybody's) (1965)
+  Aftermath (1966)
+  Between the Buttons (1967)
+  Their Satanic Majesties Request (1967)
+  Beggars Banquet (1968)
+  Let It Bleed (1969)
+  Sticky Fingers (1971)
+  Exile on Main St. (1972)
+  Goats Head Soup (1973)
+  It's Only Rock 'n Roll (1974)
+  Black and Blue (1976)
+  Some Girls (1978)
+  Emotional Rescue (1980)
+  Tattoo You (1981)
+  Undercover (1983)
+  Dirty Work (1986)
+  Steel Wheels (1989)
+  Voodoo Lounge (1994)
+  Bridges to Babylon (1997)
+  A Bigger Bang (2005)
+  Blue & Lonesome (2016)
+  `,
+  photos: (
+    <DemoLayout>
+      <img
+        alt="Mick Taylor on guitar, 1972"
+        src="https://upload.wikimedia.org/wikipedia/commons/8/8f/Mick_Taylor2.jpg"
```

Comment:
Is it okay to just link to these images?

Reply:
I much prefer the actual image, but I applied some basic layout so they sit in the panel nicely.